pages/api/capture-event.js

```diff
@@ -28,7 +28,7 @@ export default withContainer(
       return;
     }
 
-    if (!body.visitId || isNaN(body.visitId)) {
+    if (body.visitId === null || body.visitId === "" || isNaN(body.visitId)) {
```

Comment:
Would this cover visitId being undefined?

Reply:
Yush, definitely now that we've dropped worrying about a 0 visitId.